acceptance_tests/features/steps/view_created_ce_state.py

```diff
@@ -0,0 +1,37 @@
+from behave import given, when, then
+
+from acceptance_tests import browser
+from acceptance_tests.features.pages import collection_exercise, collection_exercise_details
+from config import Config
+from controllers.collection_exercise_controller import get_survey_collection_exercises
+
+
+@given('the collection exercises for that survey exist')
+def ce_exist_for_survey():
+    bricks_id = 'cb8accda-6118-4d3b-85a3-149e28960c54'
+    collection_exercises = get_survey_collection_exercises(bricks_id)
+    assert len(collection_exercises) != 0
+
+
+@when('the internal user navigates to survey details page')
+@given('the internal user is on the survey details page')
+def view_survey_details():
+    collection_exercise.go_to('Bricks')
+
+
+@then('the status of a collection exercise is Created')
+def survey_ce_state_is_created(context):
+    for row in context.table:
```

Comment:
This should be getting the table from the webpage to assert the table (from the feature file) against